package.json

```diff
@@ -1,14 +1,31 @@
 {
-  "name": "@erc6900/reference-implementation",
-  "version": "0.8.0-rc.5",
+  "name": "erc6900-reference-implementation",
```

Comment:
I tired the original name first, but got
```
npm notice Publishing to https://registry.npmjs.org/ with tag latest and public access
npm ERR! code E404
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/@erc6900%2freference-implementation - Not found
npm ERR! 404 
npm ERR! 404  '@erc6900/reference-implementation@0.8.0-rc.6' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Reply:
yeah it'd be great if we can get erc6900 back. I tried [user name and org name](https://docs.npmjs.com/creating-and-publishing-scoped-public-packages), both unavailable..